x-pack/plugins/files/server/blob_storage_service/adapters/es/content_stream/content_stream.ts

```diff
@@ -5,15 +5,24 @@
  * 2.0.
  */
 import cuid from 'cuid';
+
 import { errors as esErrors } from '@elastic/elasticsearch';
-import type { IndexRequest } from '@elastic/elasticsearch/lib/api/types';
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
 import { ByteSizeValue } from '@kbn/config-schema';
 import { defaults } from 'lodash';
 import { Duplex, Writable, Readable } from 'stream';
 
 import type { FileChunkDocument } from '../mappings';
 
+// TODO: cbor-x uses ESM modules and attempts to load a dependency when imported.
+//       When running in a Jest test environment detects "window" and causes
+//       a warning to be printed to the console: "For browser usage, directly use cbor-x/decode or cbor-x/encode modules..."
+//       we should clean this up.
+//       Also opened the following issue: https://github.com/kriszyp/cbor-x/issues/36
+// TODO: use import once type declarations for the lib are fixed
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const cborx = require('cbor-x');
+
```

Comment:
nit: Maybe to avoid this lengthy comment here, we can move this comment to a new, `cbor.ts` file, something like:

```ts
// TODO: cbor-x uses ESM modules and attempts to load a dependency when imported.
//       When running in a Jest test environment detects "window" and causes
//       a warning to be printed to the console: "For browser usage, directly use cbor-x/decode or cbor-x/encode modules..."
//       we should clean this up.
//       Also opened the following issue: https://github.com/kriszyp/cbor-x/issues/36
// TODO: use import once type declarations for the lib are fixed
// eslint-disable-next-line @typescript-eslint/no-var-requires
export const encode: SomeType1 = require('cbor-x').encode;
export const decode: SomeType2 = require('cbor-x').decode;
```